modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
@wing328 did you load the code in an IDE or run via `compile-mono.sh`? The script performs the same recursive glob that was removed from the project file, so the script would still work as expected.